src/webapp/forms.py

```diff
@@ -132,8 +132,11 @@ class GenerateResourceGroupDowntimeForm(FlaskForm):
     ])
     description = StringField("Description (the reason and/or impact of the outage)", [InputRequired()])
 
-    start_datetime = DateTimeLocalField("Local Start Datetime", format='%Y-%m-%dT%H:%M', validators=[InputRequired()])
-    end_datetime = DateTimeLocalField("Local End Datetime", format='%Y-%m-%dT%H:%M', validators=[InputRequired()])
+    start_date = DateField("Local Start Date", validators=[InputRequired()])
+    start_time = TimeField("Local Start Time", validators=[InputRequired()])
+    end_date = DateField("Local End Date", validators=[InputRequired()])
+    end_time = TimeField("Local End Time", validators=[InputRequired()])
+
```

Comment:
So do these new DateField/TimeField replacements have their own built-in formatting / validation?  (I see there is no longer a `format` parameter)

Reply:
>the browser provides an input ui

Cool